contents/handbook/company/team/tim-glaser.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Tim Glaser's README
+sidebar: Handbook
+showTitle: true
+---
+
+This guide might be helpful in working with me.
+
+## Bio
+
+I've been coding since I've been 11, which isn't as long ago as I'd like it to be. Someone first paid me to write code when I was 13 (though I'm sure they regretted it) and someone else gainfully employed me when I was 16.
+
+Originally from the Netherlands, though I quickly moved to London (I do not generally enjoy nice weather) where I joined Arachnys and shortly met James Hawkins. I went from being a software engineer, to product manager, to "leading" an R&D team, which consisted of just me.
+
+After four years I thought it was time to go do something else and had lined up a new job. Roughly 37 seconds after it was announced James wanted to "grab a beer." While plying me with alcohol, he convinced me to give up this fancy new job and instead start a startup with him.
+
+In my 'spare' time, I fall down snowy mountains, wrestle in the mud over an egg-shaped ball and watch a lot of Bondi beach in order to perfect my Australian accent.
+
+## Areas of responsibility
+
+- Make sure we move fast
+- Make sure the engineering team has all the context it needs
+- Make sure the engineering team is happy
+- Make sure we're building in the right direction
+- Sounding board to James Hawkins (CEO)
+
+## Quirks
+
+- I come out with opinions quickly and strongly. They are actually weekly held so please push back when you disagree, I'll take it well.
+- I will likely talk way too much. Please just interject/talk over me.
+- A lot of what I say sounds like a definitive statement ("this is what we're going to do") when I actually mean "here's something we could do".
+- I'm easily distracted in meetings and can come across as disinterested. It's something I'm working on but if you notice I'm drifting off please mention it.
+- Weekends are holy to me and I'll almost never work on a weekend (and don't expect anyone else to!). I also don't like to have meetings after ~8pm my time, or before 10-11am.
+- Please don't message me "hey" and then spend 3 minutes typing your question. Adding those two together is fine.
+- I like my meetings clustered together.
+
+## What I value
+
+- People who get things done without me having to chase. You're golden if I mention something off hand and it gets done.
```

Comment:
I've found that this value is more related to being able to autonomously explore/determine what's important to pursue. The current phrasing sounds as if the person working with you should be focused on small details

Reply:
There's two things here. One is ownership, the other is the thing that you're great at which is, you and I will discuss something needs to get done and I just know there will be a PR doing that thing the next day or so. I never have to chase you or check whether you've done the thing. I value that a lot.